test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,94 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, we can also run the individual steps as follows: 
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc. EC2 region, numbers of master nodes and worker nodes, and instance type can also be specified in playbook `vars`.
+
+    ```bash
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2"
+    ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 aws_key_name=mykey  num_masters=3 num_workers=3 instance_type=t2.xlarge arch=x86_64 
+    ```
+
+2. Deploy Kubernetes clusters:
+
+    ```bash
+    ansible-playbook -i .data/hosts_hub deploy-master.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ansible-playbook -i .data/hosts_hub deploy-worker.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
```

Comment:
These playbooks each deploy multiple masters/workers, in general. So the names of the playbooks should use the plural forms: "masters" and "workers".

Reply:
Fixed (A) & (B)